server/config/releases.exs

```diff
@@ -10,6 +10,12 @@ config :dev_tools, DevTool.Repo,
   database: System.get_env("POSTGRES_DB"),
   hostname: System.get_env("POSTGRES_HOST")
 
+config :application_runner, ApplicationRunner.Repo,
```

Comment:
You had to duplicate this config ?

Reply:
Yes it seems that the config contained in application-runner is not applied. That is why I added the config here.